src/styles/app-button/AppButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { useButton } from "@react-aria/button";
+import { useRef } from "react";
+
+import classnames from "classnames";
+
+import "@/styles/app-button/AppButton.scss";
+import { AppButtonProps } from "@/styles/app-button/AppButton.types";
+
+const AppButton = function ({
```

Comment:
Component should also accept ref (use `forwardRef`)

Reply:
You can use `useObjectRef`:
https://react-spectrum.adobe.com/react-aria/useObjectRef.html

Reply:
Thanks! Fixed
![image](https://github.com/user-attachments/assets/0a0a773b-a8a7-43df-8515-c1bbd517f64d)